### PR TITLE
MXNet operator initial version

### PR DESCRIPTION
**Pull request by suleisl2000**
Including
1) Source code
2) Prow config file

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/mxnet-operator/1)

### CODE REVIEW

File: pkg/controller/controller.go [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+	mxv1alpha1 "github.com/kubeflow/mxnet-operator/pkg/apis/mxnet/v1alpha1"
+	mxjobclient "github.com/kubeflow/mxnet-operator/pkg/client/clientset/versioned"
+	kubeflowscheme "github.com/kubeflow/mxnet-operator/pkg/client/clientset/versioned/scheme"
+	informers "github.com/kubeflow/mxnet-operator/pkg/client/informers/externalversions"
+	listers "github.com/kubeflow/mxnet-operator/pkg/client/listers/kubeflow/v1alpha1"
+	"github.com/kubeflow/mxnet-operator/pkg/trainer"
+)
+
+const (
+	controllerName = "kubeflow"
```
Comment (gaocegege) [inline pkg/controller/controller.go]: The name should be mxnet-operator

File: pkg/controller/controller.go [hunk 2]
```diff
@@ -0,0 +1,317 @@
...
+	MaxJobBackOff = 360 * time.Second
+)
+
+// Controller is structure to manage various service clients
+type Controller struct {
+	KubeClient  kubernetes.Interface
+	MXJobClient mxjobclient.Interface
+
+	config mxv1alpha1.ControllerConfig
+	jobs   map[string]*trainer.TrainingJob
```
Comment (gaocegege) [inline pkg/controller/controller.go]: It is not recommended in the community, we should not maintain any in-memory state in the operator.

**Comment by gaocegege** [comment]
Since it is the first commit, I think we could merge it ASAP and update it iteratively.

**Comment by gaocegege** [comment]
@suleisl2000 

Please leave a comment `/approve` to let tide merge the PR automatically

**Comment by jlewi** [comment]
The OWNERs file was misconfigured; I fixed it; please rebase off master and hopefully you can approve your changes now.

**Comment by gaocegege** [comment]
@suleisl2000 Please rebase and `/approve` to let tide merge the PR automatically.

**Pull request merged**